metricbeat/metricbeat.full.yml

```diff
@@ -275,6 +275,13 @@ metricbeat.modules:
   period: 10s
   hosts: ["kube-state-metrics:8080"]
 
+# Kubernetes events
+- module: kubernetes
+  metricsets:
+    - event
+  kube_config: ${HOME}/.kube/config
```

Comment:
* Is `${HOME}` always set in these environments?
* You left out the hosts: [...]  part. Is that on purpose?